add-on/chrome/modules/tabFrame.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/* global sendAsyncMessage */
```

Comment:
`addEventListener` is something in the eslint built-in configuration (I think as a result of telling it the code is loaded into a browser), we don't specify it as a global anywhere - I didn't really want to add it very globally, to avoid possible leaks/incorrect usage elsewhere.